### DISCUSSION TITLE
KICS 1.6 feature wishlist

### DISCUSSION THREAD

**Discussion by kaplanlior** [Checkmarx/kics#5240] [Ideas]
Hi KICS users & contributors,

KICS 1.5.0 was released about ~3 months ago, and we're in the process of finalizing the plans for KICS 1.6.0.
The goal of this thread is to raise & discuss feature requests and change you're interested in.

Previously opened ["feature request issues"]( https://github.com/Checkmarx/kics/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+request%22 )

Kaplan

**Comment by Mikisharon** [comment]
Hi KICS users & contributors,
A candidate feature for KICS 1.6 is adding support for Terraform AWS Provider version 4.0, so would like to know your votes on that.

Miki Sharon (Checkmarx Engines Product Manager)
